### ISSUE TITLE
More training details

### ISSUE THREAD

**Issue by AIlearner-cmd** [ewrfcas/Face-Super-Resolution#14]
Hi, your result is very impressive but I can not reproduce it. Is it possible that you can clarify the training details more specifically? 

From the paper, there are two steps to train. One is to train without GAN and the other is to train with GAN. What are your two command line for these two steps and how long do you train for the first step?(The command line includes a lot of parameters like your lr and D_update_ratio and so on) Thank you very much.

**Comment by ewrfcas** [comment]
https://github.com/ewrfcas/Face-Super-Resolution/blob/e83ed0bb1165ad4d916a969b5d94b8c1685c9613/models/SRGAN_model.py#L122
Here is the main train step. But I think you should check your data and preprocession first.

**Comment by AIlearner-cmd** [comment]
Thank you for reply. I use the same data as yours(ffhq and celeb) and same preprocess(resize the image with low quality). 
I am wondering how many iterations did you train for first step without GAN. Or did you just train with GAN and ignore the first step?
Also, did you just use the default parameters in train.py? If not, what are the parameters you set for first step and second step? There should be some difference like the weight of three kinds of loss.
Thank you again.

**Comment by ewrfcas** [comment]
I haven't trained this project according to the paper. It just works as the code here. There is only one step for training with ESRGAN.

**Comment by HITxyer** [comment]
Thank you for your sharing,  Could you tell us about your training data? Looking forward to your reply.

**Comment by ewrfcas** [comment]
You can find the preprocess in gen_lr_imgs.py. The training data is ffhq and celeba-hq
